vitest.config.ts

```diff
@@ -81,14 +81,14 @@ export default defineConfig(() =>
           'jest-extended/all',
           'expect-more-jest',
           './test/setup.ts',
-          './test/jest-legacy.ts',
           'test/to-migrate.ts',
         ],
         reporters: ci ? ['default', 'github-actions'] : ['default'],
         mockReset: true,
         coverage: {
           provider: 'v8',
           ignoreEmptyLines: true,
+          skipFull: true,
```

Comment:
Are they still contained in coverage lcov and json files?

Reply:
not sure, so maybe `!CI` here?

Reply:
codecov is happy, no coverage changes, so it's ok to set it

Reply:
doesn't matter on ci too, as we don't show the files 😏